edgelet/contrib/config/linux/config.yaml

```diff
@@ -150,6 +150,7 @@ provisioning:
 #   device_ca_cert: "<ADD URI TO DEVICE CA CERTIFICATE HERE>"
 #   device_ca_pk: "<ADD URI TO DEVICE CA PRIVATE KEY HERE>"
 #   trusted_ca_certs: "<ADD URI TO TRUSTED CA CERTIFICATES HERE>"
+#   auto_generated_ca_lifetime_days: "<Default is 90 days>"
```

Comment:
`"<Default is 90 days>"` implies the value is a string `"90"` or even `"90 days"`, rather than a number `90`.

But anyway, it's probably better to mention it in the comment block at lines 134-141:

```
#     auto_generated_ca_lifetime_days - The lifetime of the auto-generated workload CA certificate.
                                        If device_ca_cert and device_ca_pk have not been set,
                                        then this also applies to the auto-generated device CA certificate.
                                        Defaults to 90 days.
```

and change this one to `# auto_generated_ca_lifetime_days: <value>`

Same for `windows/config.yaml`